docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -752,6 +752,10 @@ To collect statistics for a table, execute the following statement::
 
   ANALYZE table_schema.table_name;
 
+To recalculate from scratch the statistics for the table use additional parameter ``full_refresh``.
+
+  ANALYZE table_schema.table_name WITH(full_refresh = true);
```

Comment:
How about changing to enum (`mode` or something) instead of boolean? Are we sure that we don't add new mode in the future?